torch/_dynamo/debug_utils.py

```diff
@@ -853,9 +853,8 @@ def dynamo_minifier_backend(gm, example_inputs, compiler_name):
     except Exception as exc:
         orig_failure = str(exc)
         log.warning(
-            "Compiled Fx GraphModule failed with following error. Starting minifier."
+            "Compiled Fx GraphModule failed. Creating script to minify the error."
```

Comment:
Will we see the failure at all? I think today, we see multiple copies of the same exception. Just confirming that we should have atleast one.

Reply:
Yeah we'll see it once, it's bubbled up to the top level in eval_frame.py

Reply:
Perfect